<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>180</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Drink</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Status Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>255</width>
        <height>195</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Status Menu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position 6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>280</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>250</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>20</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>80</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>80</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingredient</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Refill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
